examples/tutorials/02-parameterization/sensor-05.yaml

```diff
@@ -5,6 +5,10 @@ metadata:
 spec:
   template:
     serviceAccountName: operate-workflow-sa
+    container:
```

Comment:
?

Reply:
sorry, accidentally committed the DEBUG_LOG setting as well. I'll remove that